MementoMori/MementoMori.Server/Controllers/DeckController.cs

```diff
@@ -144,14 +144,28 @@ public async Task<ActionResult> DeleteDeck(Guid deckId)
             var requesterId = _authService.GetRequesterId(HttpContext);
             try
             {
-                await _deckHelper.DeleteDeckAsync(deckId, (Guid)requesterId);
-                return Ok();
+                if(requesterId != null)
+                {
+                    await _deckHelper.DeleteDeckAsync(deckId, (Guid)requesterId);
+                    return Ok();
+                }
+                else
+                    return Unauthorized();
+
             }
             catch
             {
                 return StatusCode(500);
             }
         }
+        [HttpGet("hasAccessIfPrivate")]
```

Comment:
Would be checked in methods that return deck info anyways, any reason to have this?

Reply:
Removed.